### PR TITLE
Fix shadowed vars pt6

### PR DESCRIPTION
**Pull request by pugnascotia**
Part of #19752. Fix more instances where local variable names were shadowing field names.

### CODE REVIEW

**Comment by sethmlarson** [comment]
(The ping for clients team can be ignored, was triggered by modifying `ESRestTestCase.java`)

File: test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java [hunk 1]
```diff
@@ -114,11 +114,11 @@ public String name() {
 
         @Override
         public long getTotalSpace() throws IOException {
-            final long totalSpace = this.totalSpace;
-            if (totalSpace == -1) {
+            final long totalSpaceCopy = this.totalSpace;
```
Comment (mark-vieira) [inline test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java:117]: I don't quite see the point is creating this intermediate variable in the first place.

**Review by mark-vieira — APPROVED** [review]
LGTM

File: test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java (hunk 1 above)
Comment (pugnascotia) [reply test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java:117]: It's a `volatile` variable, I assume the copy is something to do with that.

**Pull request merged**